packages/algob/src/internal/deployer.ts

```diff
@@ -143,34 +143,34 @@ class DeployerBasicMode {
 
   /**
    * Loads stateful smart contract info from checkpoint
-   * @param nameApproval Approval program name
-   * @param nameClear clear program name
+   * @param approvalFileName Approval program name
+   * @param clearFileName clear program name
    */
-  getApp (nameApproval: string, nameClear: string): rtypes.SSCInfo | undefined {
-    return this.checkpoint.getAppfromCPKey(nameApproval + "-" + nameClear);
+  getAppByFile (approvalFileName: string, clearFileName: string): rtypes.SSCInfo | undefined {
+    return this.checkpoint.getAppfromCPKey(approvalFileName + "-" + clearFileName);
   }
 
   /**
    * Loads stateful smart contract info from checkpoint
    * @param appName name of the app (passed by user during deployment)
    */
-  getAppByName (appName: string): rtypes.SSCInfo | undefined {
+  getApp (appName: string): rtypes.SSCInfo | undefined {
     return this.checkpoint.getAppfromCPKey(appName);
   }
 
   /**
    * Loads logic signature info(contract or delegated) from checkpoint (by lsig name)
    * @param lsigName name of the lsig (passed by user during mkContractLsig/mkDelegatedLsig)
    */
-  getLsigByName (lsigName: string): LogicSigAccount | undefined {
-    return this.getDelegatedLsig(lsigName) ?? this.getContractLsig(lsigName);
+  getLsig (lsigName: string): LogicSigAccount | undefined {
```

Comment:
actually `getDelegatedLsig` works for both -- byFile (eg. `file.py`), or byname -- (eg. "myLsig").

I renamed them to "byFile" so that users will use a single function `getLsig` for names, which would be easy to understand. We can deprecate `getDelegatedLsigByFile`, `getContractLsigByFile` functions, but then again it creates a confusion for `getLsig` function (as now from the user's perspective it supports both file and lsig)